src/NuGet.Server/Core/DefaultServiceResolver.cs

```diff
@@ -44,8 +53,12 @@ public object Resolve(Type type)
                 return _packageAuthenticationService;
             }
 
-
             return null;
         }
+
+        public void Dispose()
+        {
+            _packageRepository.Dispose();
+        }
```

Comment:
This class isn't intended to be subclassed, so the `Disposing` method isn't needed. I'll make the class `sealed`.